selfdrive/car/hyundai/carstate.py

```diff
@@ -60,15 +58,20 @@ def get_can_parser(CP):
 
     ("CF_Lvr_GearInf", "LVR11", 0),        #Transmission Gear (0 = N or P, 1-8 = Fwd, 14 = Rev)
 
-    ("CR_Mdps_DrvTq", "MDPS11", 0),
```

Comment:
MDPS11 is not active (present, but always 0) on some older Hyundai.
As such, they cannot detect a user touching the wheel.
This message is also in MDPS12, and is present on all vehicles.